src/annotator/adder.js

```diff
@@ -192,6 +195,9 @@ export class Adder {
     // and close to the end.
     const hMargin = Math.min(ARROW_H_MARGIN, selectionRect.width);
     const adderWidth = this._width();
+    // Render the adder a little lower on a touch device to provide room for the
+    // selection grab handles.
+    const touchScreenMargin = isTouchDevice() ? 10 : 0;
```

Comment:
```suggestion
    const touchScreenOffset = isTouchDevice() ? 10 : 0;
```

"Offset" seems to me like a more appropriate term here than margin since we're talking about a position (the Y coordinate of the top of the adder) rather than a box (which has a margin around it).

Reply:
agreed